### PR TITLE
add sample IAM permission

### PR DESCRIPTION
**Pull request by M00nF1sh**
add sample IAM permission needed for the controller while Restricted the permissions where ever possible.

TODO: the permissions can be further scoped down when the vpcID / clusterName can be provided, we should document how to further scope down permissions.

### CODE REVIEW

**Comment by astrived** [comment]
Do we have write up on how to further scope down when vpcID / clusterNam ?

File: docs/install/iam_policy.json [hunk 1]
```diff
@@ -0,0 +1,226 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "iam:CreateServiceLinkedRole",
+                "ec2:DescribeAccountAttributes",
+                "ec2:DescribeAddresses",
+                "ec2:DescribeInternetGateways",
```
Comment (astrived) [inline docs/install/iam_policy.json:10]: Do we need IGW Describe ?

Comment (M00nF1sh) [reply docs/install/iam_policy.json:10]: > Do we have write up on how to further scope down when vpcID / clusterName are provided ?

no, it's a TODO item.

Comment (M00nF1sh) [reply docs/install/iam_policy.json:10]: > Do we need IGW Describe ?

It's need by ELB implicitly during CreateLoadBalancer call: https://docs.aws.amazon.com/elasticloadbalancing/latest/userguide/elb-api-permissions.html

Comment (astrived) [reply docs/install/iam_policy.json:10]: LGTM

**Review by astrived — APPROVED** [review]

File: docs/install/iam_policy.json [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+                "elasticloadbalancing:ModifyTargetGroup",
+                "elasticloadbalancing:ModifyTargetGroupAttributes",
+                "elasticloadbalancing:RegisterTargets",
+                "elasticloadbalancing:DeregisterTargets",
+                "elasticloadbalancing:DeleteTargetGroup"
+            ],
+            "Resource": "*",
+            "Condition": {
+                "Null": {
+                    "aws:ResourceTag/ingress.k8s.aws/cluster": "false"
```
Comment (kishorj) [inline docs/install/iam_policy.json:190]: How does backwards compatibility work? Do we recommend these IAM permissions for new setups only?

**Review by kishorj — APPROVED** [review]

**Pull request merged**